templates/pipelines/common-app-ci.yaml

```diff
@@ -7,6 +9,11 @@ parameters:
 - name: deployConfigOnly
   type: boolean
   default: false   
+- name: generateFluxManifest
+  type: boolean
+  default: false
+- name: fluxManifestApiBaseUri
```

Comment:
is this parameter needed?

Reply:
We need the Api Url to be passed in to the script.

Reply:
Removed